measures/ResStockArguments/measure.rb

```diff
@@ -427,20 +421,20 @@ def run(model, runner, user_arguments)
     end
 
     # Num Occupants
-    if args['geometry_unit_num_occupants'] == Constants.Auto
+    if args['geometry_unit_num_occupants'].to_s == Constants.Auto
       args['geometry_unit_num_occupants'] = Geometry.get_occupancy_default_num(args['geometry_unit_num_bedrooms'])
     else
-      args['geometry_unit_num_occupants'] = Integer(args['geometry_unit_num_occupants'])
+      args['geometry_unit_num_occupants'] = Integer(args['geometry_unit_num_occupants'].to_s)
```

Comment:
Why string before integer?

Reply:
It used to be a [required string argument](https://github.com/NREL/OpenStudio-HPXML/blob/ac525ddca5d5178e089bb1104e86edc59ddbde4e/BuildResidentialHPXML/measure.rb#L260). Now it's an [optional double argument](https://github.com/NREL/resstock/blob/d0cdc51e9f4c9dc91b3af5727804c097073edbf0/resources/hpxml-measures/BuildResidentialHPXML/measure.rb#L257) (which is converted to an optional string argument to support "auto"), so I use `.to_s` to convert from optional string to string. I could have done `.get` instead.